### PR TITLE
Direct debit payments.

### PR DESCRIPTION
**Pull request by duozersk**

### CODE REVIEW

File: modules/features/falcon_feature_commerce_direct_debit_gateway/falcon_feature_commerce_direct_debit_gateway.info.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: 'Falcon Commerce Direct Debit Gateway feature'
```
Comment (kalabro) [inline modules/features/falcon_feature_commerce_direct_debit_gateway/falcon_feature_commerce_direct_debit_gateway.info.yml:1]: Could you please explain the value of this feature? From what I see it just enables one contrib module https://www.drupal.org/sandbox/duozersk/3005285 and that's it. I'd enable it directly in the client project just like we did for Realex (tests can live in client project too). What do you think?

File: modules/features/falcon_feature_user/config/install/user.role.appeals_manager.yml [hunk 2]
```diff
@@ -12,15 +12,19 @@ permissions:
   - 'access toolbar'
   - 'create appeal content'
   - 'create media'
+  - 'create source_code entities'
```
Comment (kalabro) [inline modules/features/falcon_feature_user/config/install/user.role.appeals_manager.yml:15]: Intentional?

File: composer.json [hunk 3]
```diff
@@ -21,6 +21,10 @@
     {
       "type": "composer",
       "url": "https://packages.drupal.org/8"
+    },
+    {
+      "type": "vcs",
+      "url": "https://git.drupal.org/sandbox/duozersk/3005285.git"
     }
```
Comment (kalabro) [inline composer.json:28]: I've got few questions about the sandbox project if I can list them here:
```
$fields['debit_date'] = BundleFieldDefinition::create('integer')
      ->setLabel(t('Debit date'))
      ->setRequired(TRUE)
      ->setSetting('size', 'tiny');
```
Could you please help me understand why do we need to store `debit_date` on payment method level? Can it be stored on payment gateway config level? In this case we won't need to pass this data from frontend each time.

I can see accept_direct_debits and one_signatory for both SEPA and UK. Do we need those for UK?

File: modules/features/falcon_feature_user/config/install/user.role.appeals_manager.yml (hunk 2 above)
Comment (duozersk) [reply modules/features/falcon_feature_user/config/install/user.role.appeals_manager.yml:15]: yes

File: modules/features/falcon_feature_commerce_direct_debit_gateway/falcon_feature_commerce_direct_debit_gateway.info.yml (hunk 1 above)
Comment (duozersk) [reply modules/features/falcon_feature_commerce_direct_debit_gateway/falcon_feature_commerce_direct_debit_gateway.info.yml:1]: It adds the payment gateways - test and live one, for the direct debits.

**Review by kalabro — APPROVED** [review]

**Pull request merged**